sdk/ai/azure-ai-resources/azure/ai/resources/operations/_connection_operations.py

```diff
@@ -30,20 +30,34 @@ def __init__(self, *, resource_ml_client: MLClient = None, project_ml_client: ML
 
     @distributed_trace
     @monitor_with_activity(logger, "Connection.List", ActivityType.PUBLICAPI)
-    def list(self, connection_type: Optional[str] = None, scope: str = OperationScope.AI_RESOURCE) -> Iterable[BaseConnection]:
+    def list(
+        self,
+        connection_type: Optional[str] = None,
+        scope: str = OperationScope.AI_RESOURCE,
```

Comment:
Are we going to rename the scope to be `AI_HUB` soon?

Reply:
I'll ask a few PMs. I hadn't heard recently that we were changing the name of AI resource back to AI Hub.